### ISSUE TITLE
Triage Dashboard Design

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#1861]
Context: #1857 
Depends on: #1858 (HTML template)

This is an area which is completely free. I was [trying to explore ideas](http://www.la-grange.net/tmp/triage.html).
@magsout feel free to propose anything.

![capture d ecran 2017-11-02 a 10 59 01](https://user-images.githubusercontent.com/505230/32306106-dd80d9ee-bfbc-11e7-8e42-a4f23c905bab.png)

**Comment by karlcow** [comment]
Some mood ideas
* https://i.pinimg.com/736x/7c/ef/1a/7cef1a6f101a14cc75729646e564f1e8--clean-design-design-ui.jpg
* https://i.pinimg.com/236x/48/38/c9/4838c9149bdd177ae197644770fa043b--data-dashboard-dashboard-design.jpg

**Comment by magsout** [comment]
Numbers are cool, the design is simplet and efficient.

For the issue's ist, it depends of https://github.com/webcompat/webcompat.com/issues/1857#issuecomment-341820798

So we can use the component Issue from webcompat or styling a new one very rapidly.

**Comment by karlcow** [comment]
The dashboard should not use the webcompat styles. 
They can even stay inline for now.

**Comment by karlcow** [comment]
@magsout so that's it. Je suis à votre service. ☕️🛠 Feel free to propose things, play, and stuff and if you need me to help you modify anything.

**Comment by magsout** [comment]
First test:

![screenshot-2017-11-17 triage dashboard webcompat com](https://user-images.githubusercontent.com/1997108/32953140-9fc24d14-cbaf-11e7-8424-d10006bb0e33.png)

**Comment by magsout** [comment]
@miketaylr the csp security blocks me to add inline javascript, how I can disabled it ?

**Comment by magsout** [comment]
![capture d ecran 2017-11-17 a 15 55 24](https://user-images.githubusercontent.com/1997108/32953201-c98d9680-cbaf-11e7-9b50-cc86b38baf74.png)

**Comment by miketaylr** [comment]
@magsout you would add `'unsafe-inline` to https://github.com/webcompat/webcompat.com/blob/master/webcompat/helpers.py#L498.

Will this dashboard require inline JS? Or is this just for dev?

If it's required, we can implement support for nonce: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#Unsafe_inline_script

**Comment by magsout** [comment]
@miketaylr 

>Will this dashboard require inline JS? Or is this just for dev?

Both

**Comment by magsout** [comment]
@karlcow if you want to take a look I pushed my work on https://github.com/webcompat/webcompat.com/tree/issues/1861

- Added CSS
- Organize Html, folders 

I have to do the JavaScript part for the selectors.

**Comment by magsout** [comment]
@karlcow  The responsive is not ended yet, Flexbox does the work, but I have to improve some things here and then

**Comment by magsout** [comment]
@karlcow I have one request plz ;) 

could you add a json function for every issue config https://github.com/webcompat/webcompat.com/commit/450620be0e258a8bfca56de375517fc71ac2a95e#diff-c4ca2145e024be83289f04bb020d1aedR71 ?

I could handle the config in JavaScript

**Comment by magsout** [comment]
@karlcow

![screen recording 2017-11-23 at 08 29 am](https://user-images.githubusercontent.com/1997108/33162701-3da7403c-d02b-11e7-8ac5-21363e6cfbd3.gif)

**Comment by miketaylr** [comment]
This was merged into dashboard branch.
